openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -116,7 +116,7 @@ void ieee154e_init(void) {
     memset(&ieee154e_dbg,0,sizeof(ieee154e_dbg_t));
 
     // set singleChannel to 0 to enable channel hopping.
-    ieee154e_vars.singleChannel     = 0;
+    ieee154e_vars.singleChannel     = 11;

```

Comment:
by default, channel hopping is enabled,  please reverse this change.

Reply:
Ok, reveresed it.

Reply:
resolve here